mysqlnd_azure.c

```diff
@@ -410,19 +436,32 @@ MYSQLND_METHOD(mysqlnd_azure_data, connect)(MYSQLND_CONN_DATA ** pconn,
                 port = ui_redirect_port;
                 transport = redirect_transport;
```

Comment:
here transport and redirect_transport are point to same char*. I suggest we set redirect_transport.s to null here as we don't use redirect_transport later if succeeded. this is will help to avoid someone free redirect_transport.s again which cause AV when we access transport .s

Reply:
added. Thanks.